spec/systems/keywords/list_keywords_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'View keywords list', type: :system do
+  context 'given NO keyword' do
+    it 'displays a no-keyword notice and DO NOT displays the pagination nav' do
+      Keyword.delete_all
+
+      visit(root_path)
+
+      expect(page).to(have_selector('[data-testid="no-keyword-notice"]'))
+      expect(page).not_to(have_selector('nav#keywords-table__pagination'))
+    end
+  end
+
+  context 'given SOME keywords' do
```

Comment:
As suggested in another PR:

```suggestion
  context 'given a list of keywords' do
```

Reply:
Fixed in 4c34b3e07c7d5c18ff6255687b4f66ae63c55737